tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -319,22 +312,23 @@ absl::optional<ConvWithConvertOrClamp> FindConvWithClamp(
   using match::Op;
 
   // The pattern we want to match:
-  //   clamp(broadcast(-128), (get_tuple_element(custom_call(int8_x,
-  //   int8_w, ...)), broadcast(127);
+  //   convert<int8>(clamp(broadcast(-128), (get_tuple_element(custom_call(int8_x,
```

Comment:
Since only int8 output needs clamp and it must come with convert<int8>, the pattern includes both of them.